AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -78,6 +78,8 @@ astra-honda.com,twist.com.tr,publi24.ro,bauernzeitung.ch,cerrahi.com.tr,lescon.c
 butlers.com,stiefelparadies.de,sturcookware.de,paradies-bettenshop.de,modbrix.de,droniq.de,hallesaale.shop,ossiladen.de,autoscheibenwischer24.com,objekteunserertage.com,schuhbeck.de,direktvomfeld.eu,kliemannsland.de,lanakk.com,catalea.de,mm-germany.com,rollei.de,oderso.cool,shop.autorenwelt.de,healthmask.de###banner-wrapper
 ! ###cc-banner-wrap
 serien.de,blumen-objekte.de,soldat-und-technik.de,timschropp.com,gfc-gruppe.de,literaturcafe.de,dashdot.de,appoid.de,rafalabrador.com###cc-banner-wrap
+! #tt-cookie-alert (Teamtailor)
+career.kry.se,teamtailor.com###tt-cookie-alert
```

Comment:
Is it good idea to move these into `cookies_general`?

Reply:
I think so, used by 100 websites: `https://publicwww.com/websites/%22tt-cookie-alert%22/`

Reply:
Pushed at https://github.com/AdguardTeam/AdguardFilters/pull/102899/commits/e91400418032b51f66abb5fbdb2ca80293823689. Thanks you.